packages/twenty-front/src/modules/object-metadata/schemas/camelCaseStringSchema.ts

```diff
@@ -0,0 +1,8 @@
+import camelCase from 'lodash.camelcase';
```

Comment:
I'm not against introducing a new folder in our module hiearchy. However, I think "schemas" is not clear here. How does this play with types? it feels a bit duplicates, should we merge them both in the same folder and make schemas a subfolder? We are missing some vision here

Also this folder i mixing some generic utils (camelCaseStringSchema.ts) and some business related schemas